pkg/kubelet/types/pod_update.go

```diff
@@ -34,34 +35,38 @@ const (
 // PodOperation defines what changes will be made on a pod configuration.
 type PodOperation int
 
+// These constants identify the PodOperations that can be made on a pod configuration.
 const (
-	// This is the current pod configuration
+	// SET is the current pod configuration.
 	SET PodOperation = iota
-	// Pods with the given ids are new to this source
+	// ADD: Pods with the given ids are new to this source.
 	ADD
-	// Pods with the given ids are gracefully deleted from this source
+	// DELETE: Pods with the given ids are gracefully deleted from this source.
```

Comment:
I don't really like how you approached it with adding semicolon. Would love to see more human readable thing. But it's fine.

Reply:
I've updated the comments to be more human readable. Please take a look.